Ginger/Ginger/UserControlsLib/TextEditor/ValueExpression/ValueExpressionMockDataEditorPage.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<Page x:Class="Ginger.UserControlsLib.TextEditor.ValueExpression.ValueExpressionMockDataEditorPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+      xmlns:local="clr-namespace:Ginger.UserControlsLib.TextEditor.ValueExpression"
+      xmlns:GingerCore="clr-namespace:GingerCore;assembly=GingerCore"    
+      mc:Ignorable="d" 
+      d:DesignHeight="300" d:DesignWidth="300"
+       Title="ValueExpressionMockDataEditorPage">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="3*"/>
+            <RowDefinition Height="15*"/>
+        </Grid.RowDefinitions>
+
+        <Label x:Name="lbl" Grid.Row="0" Style="{StaticResource $LabelStyle}">
+            <GingerCore:ucTextDicResource Text="Select Alternative Functions:"/>
+        </Label>
+        <ListView x:Name="FunctionsList" Grid.Row="1" Style="{StaticResource $ListBoxStyle}" Foreground="{StaticResource $SelectionColor_Pink}" BorderThickness="0">
```

Comment:
Check if tool tip can be added to List View item and try to show function summary in it

Reply:
we do it in next PR